### PR TITLE
Bug/6.3/tdi 37658 runtime

### PR DESCRIPTION
**Pull request by wwang-talend**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
tsalesforcebulkexec can't work in studio

**What is the new behavior?**
now it can work

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://jira.talendforge.org/browse/TDI-37658

### CODE REVIEW

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/340/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/340/)

File: components-salesforce/src/test/java/org/talend/components/salesforce/test/SalesforceComponentTestIT.java [hunk 1]
```diff
@@ -719,8 +724,24 @@ public void testRuntimeInfo() {
         ComponentProperties properties = this.getComponentService()
                 .getComponentProperties(TSalesforceBulkExecDefinition.COMPONENT_NAME);
 
-        assertNotNull("should not null", definition.getRuntimeInfo(properties, ConnectorTopology.NONE));
-        assertNotNull("should not null", definition.getRuntimeInfo(properties, ConnectorTopology.OUTGOING));
+        RuntimeInfo outgoing = definition.getRuntimeInfo(properties, ConnectorTopology.OUTGOING);
+        RuntimeInfo none = definition.getRuntimeInfo(properties, ConnectorTopology.NONE);
+
+        assertNotNull("should not null", outgoing);
```
Comment (gonchar-ivan) [inline components-salesforce/src/test/java/org/talend/components/salesforce/test/SalesforceComponentTestIT.java]: you may also compare expected and actual class name returned by RuntimeInfo.getRuntimeClassName()

Comment (wwang-talend) [inline components-salesforce/src/test/java/org/talend/components/salesforce/test/SalesforceComponentTestIT.java]: The statement contains that : 
assertTrue("the type is not right", instance.getClass() == SalesforceBulkExecSource.class);

**Comment by wwang-talend** [comment]
there is a limit for the pr : don't support the nb line and success line and reject line, so close it.

**Pull request closed without merging**